src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -1229,6 +1284,32 @@ public MouseInOut MouseInOut
             }
         }
 
+        /// <summary>
+        /// DeviceInfo evnet arguments.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class DeviceInfoEventArgs : EventArgs
+        {
+            private DeviceInfo deviceInfoEvent;
+
+            /// <summary>
+            /// DeviceInfo event.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public DeviceInfo DeviceInfo
+            {
+                get
+                {
+                    return deviceInfoEvent;
+                }
+                set
```

Comment:
IMO, I don't think it needs to provide public setter.
How about make constructor with an argument instead?
```C#
public class DeviceInfoEventArgs : EventArgs
{
    public DeviceInfoEventArgs(DeviceInfo deviceInfo)
    {
        DeviceInfo = deviceInfo;
    }

    public DeviceInfo DeviceInfo
    {
        get;
    }
}
```